README.md

```diff
@@ -147,6 +145,32 @@ You may also achieve above 5-7 via Coral-Service simple UI: http://localhost:808
 ```  
 7. Translate a query on existing table/view in remote metastore using the /translate endpoint
 
+### Coral Service UI
+Apart from the CLI command above, you can also interact with Coral Service via a simple UI.
+
+After running `../gradlew bootRun` or `../gradlew bootRun --args='--spring.profiles.active=localMetastore'` in coral-service module, 
+the UI can be accessed by http://localhost:8080:
```

Comment:
After running `../gradlew bootRun` (for remote metastore mode) or `../gradlew bootRun --args='--spring.profiles.active=localMetastore'` (for local metastore mode) from coral-service module, 
the UI can be accessed from the browser. Use the URL `http://localhost:8080:` to run the UI on a local browser.

Reply:
Modified, thanks.